tests/kernel/pipe/pipe_api/src/test_pipe_fail.c

```diff
@@ -110,4 +112,190 @@ void test_pipe_user_get_fail(void)
 
 	get_fail(p);
 }
+
+/**
+ * @brief Test k_pipe_alloc_init() failure scenario
+ *
+ * @details Verify that the parameter of the API is
+ * not initialized, what will happen.
```

Comment:
It would better to change it like below.

"See what will happen if an uninitialized k_pipe is passed to k_pipe_all_init()."

Also, please check "@detail" of the others functions.

Reply:
Hi @LeiW000 , I have updated the comments.